Entities/AtomGroup.h

```diff
@@ -143,11 +143,18 @@ namespace RTE {
 		int GetResolution() const { return m_Resolution; }
 
 		/// <summary>
-		/// Gets The depth Atoms in this AtomGroup are placed off the edge of the owning MOSR's graphical representation outline towards it's center.
+		/// Gets the depth Atoms in this AtomGroup are placed off the edge of the owning MOSR's graphical representation outline towards it's center.
 		/// </summary>
 		/// <returns>The depth, in pixels. If 0, Atoms are placed right on the edge of the MOSR outline.</returns>
 		int GetDepth() const { return m_Depth; }
 
+		/// <summary>
+		/// Gets the position of an Atom in this AtomGroup adjusted to the Owner MOSRotating horizontal flip and rotation.
+		/// </summary>
+		/// <param name="atom">The individual Atom to get the position for.</param>
+		/// <returns>The position of an Atom in this AtomGroup adjusted to the Owner MOSRotating horizontal flip and rotation.</returns>
+		Vector GetAtomPos(const Atom *atom) const;
```

Comment:
I would rename this to GetRotatedAtomOffset or something. Position tends to be used for scene position while offset tends to be used for position in relation to the owner, which this is.